tests/functional/test_install_reqs.py

```diff
@@ -233,7 +237,7 @@ def test_wheel_user_with_prefix_in_pydistutils_cfg(
         user_filename = ".pydistutils.cfg"
     else:
         user_filename = "pydistutils.cfg"
-    user_cfg = os.path.join(os.path.expanduser('~'), user_filename)
+    user_cfg = os.path.join(get_home(), user_filename)
```

Comment:
This change isn't necessary - the original code is fine.

Reply:
Extended logic to other functions as well.